hub/management/commands/import_new_constituencies.py

```diff
@@ -0,0 +1,49 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+import pandas as pd
+from tqdm import tqdm
+
+from hub.models import Area, AreaType
+
+
+class Command(BaseCommand):
+    help = "Import basic area information for new constituencies"
+
+    data_file = settings.BASE_DIR / "data" / "new_constituencies.csv"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-q", "--quiet", action="store_true", help="Silence progress bars."
+        )
+
+    def handle(self, quiet: bool = False, *args, **options):
+        if not quiet:
+            print("Importing Areas")
+        df = pd.read_csv(self.data_file)
+        area_type, created = AreaType.objects.get_or_create(
+            name="2023 Parliamentary Constituency",
```

Comment:
It looks like we’ve made a decision, elsewhere (eg: https://pages.mysociety.org/2025-constituencies/) to call these "2025" constituencies. So should we use `2025 Parliamentary Constituency` and `WMC25` here? /cc @ajparsons

Reply:
FWIW I discussed this with @ajparsons and we think we should refer to these new constituencies as either "2025 constituencies" or "Future constituencies" (if we want to hedge our bets) when we’re limited for space, and "Constituencies at the next election" if we have more space.

I see @struan has already modified `import_new_constituencies.py` to use the latter :+1: